docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -38,24 +38,12 @@ The instructions below, provide a recommended step by step guide to creating and
 
 ## Installing PyTorch Using the Terminal
 
-There are two instances of PyTorch that can be installed, one requiring GPUs, and another utilising only CPUs. GPUs generally improve project compute speeds and are preferred. For both instances of pytorch, please follow these steps;
+There are two instances of PyTorch that can be installed, one requiring GPUs, and another utilizing only CPUs. The use of GPUs improve compute speeds and are preferred. For both instances of pytorch, please follow these steps;
 
-1. [Create](../../workflow_solutions/using_anaconda.md#create-an-environment) and [activate](../../workflow_solutions/using_anaconda.md#activate-an-environment) an environment as stated in these links.
-
-1. Access the terminal following the steps [here](#installing-anaconda-environments-using-the-terminal).
-
-<!-- markdownlint-disable MD046 -->
-!!! note
-
-    When installing packages, modules and libraries into environments, remember to also install `ipykernel` using `conda install ipykernel`. This way your activated environment would appear in the list of kernels in your Jupyter Notebook.
-
-<!-- markdownlint-enable MD046 -->
-
-For a correct installation of pytorch, we have to ensure some conditions are met. See partition [docs](../hardware.md#details) for a guide. One of such conditions, is to load CUDA toolkit using the below command in your environment setup form (see image below).
+1. For a correct installation of pytorch using GPUs, we have to ensure some conditions are met. See partition [docs](../hardware.md#details) for a guide. One of such conditions, is to load the CUDA toolkit using the below command in your environment setup form (see image below).
```

Comment:
Please clarify "partition docs".

If we tell folks there are "some conditions" then we need to list them all. In a step-by-step instruction, that phrasing "some conditions" could be replaced by explicitly stating each step as they come up in the flow of steps.

I realize this was in the original, but no time like today to fix up ambiguous statements.